lib/graphhopper/api/vrp_api.rb

```diff
@@ -50,6 +50,8 @@ def post_vrp_with_http_info(key, body, opts = {})
       header_params['Accept'] = @api_client.select_header_accept(['application/json'])
       # HTTP header 'Content-Type'
       header_params['Content-Type'] = @api_client.select_header_content_type(['application/json'])
+      # HTTP header 'Content-Type'
```

Comment:
©️ ✂️ (copy-paste 😆 )

And do we event need these comments? They don't seem helpful.

Reply:
I don't think they are very useful but I find it easier to read the code when it is consistent.